setup.cfg

```diff
@@ -48,10 +48,11 @@ markers=
 filterwarnings=
     error
     ignore::pytest.PytestUnraisableExceptionWarning
-    ignore:numpy.ndarray size changed:RuntimeWarning
     # ignore benign Cython warnings on ndarray size
-    ignore::DeprecationWarning:ml_dtypes.*
-    # ignore benign ml_dtypes DeprecationWarning
+    ignore:numpy.ndarray size changed:RuntimeWarning
+    # ignore DeprecationWarnings from dependency packages
+    ignore::DeprecationWarning:(?!desc)
```

Comment:
I checked it locally (by calling a deprecated function from numpy inside of `coils.py`) and this causes that warning to propagate thru and be fiiltered as an error, so I think it does what we want (i.e. if we call directly the deprecated function it will warn us)

Reply:
but correctly does not cause an error on the deprecation warning raised inside of interpax